consensus/blake3pow/poem.go

```diff
@@ -149,6 +163,53 @@ func (blake3pow *Blake3pow) UncledLogS(block *types.WorkObject) *big.Int {
 	return totalUncledLogS
 }
 
+func (blake3pow *Blake3pow) WorkShareLogS(wo *types.WorkObject) (*big.Int, error) {
+	workShares := wo.Uncles()
+	totalWsEntropy := big.NewInt(0)
+	for _, ws := range workShares {
+		powHash, err := blake3pow.ComputePowHash(ws)
+		if err != nil {
+			return big.NewInt(0), err
```

Comment:
Nit: nil is better than 0, even though we return an err

Reply:
just returned 0 because it is an integer value and doesnt add any entropy